ext/storage_sources/gcp_store/gcp_storage_source.cpp

```diff
@@ -69,6 +71,14 @@ static int gcp_object_list_free(WT_FILE_SYSTEM *, WT_SESSION *, char **, uint32_
   __attribute__((__unused__));
 static int gcp_terminate(WT_STORAGE_SOURCE *, WT_SESSION *) __attribute__((__unused__));
 
+// Construct a pathname from the directory and the object name.
+static std::filesystem::path
+gcp_path(const std::filesystem::path &dir, const std::filesystem::path &path)
+{
+    // Skip over "./" and variations (".//", ".///./././//") at the beginning of the name.
+    return (dir / std::filesystem::canonical(path));
```

Comment:
I don't see the need to call a function which one line calls another function. Can we just directly use canonical and remove this function?